scripts/zones/Sauromugue_Champaign_[S]/mobs/Herensugue.lua

```diff
@@ -3,6 +3,11 @@
 --   NM: Herensugue
 -----------------------------------
 
+function onMobInitialize(mob)
+    mob:setMod(dsp.mod.TRIPLE_ATTACK, 90) -- "Triple Attacks almost every round"
```

Comment:
keep the mod because of the rate, but I would bet we have the same job issue on this mob as well

Reply:
nvm he's thf/thf xD